tests/cases/fourslash/todoComments19.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path='fourslash.ts' />
+
+// Tests that todos are not found in node_modules folder.
+
+// @Filename: todoTest0.ts
+//// import * as foo1 from "fake-module";
+
+// @Filename: node_modules/fake-module/ts.ts
+//// // TODO
+
+verify.todoCommentsInCurrentFile(["TODO"]);

```

Comment:
The check is for `/node_modules/` — would it apply in Windows with its backslash directory separators?

Would it be sensible to add a test for that?